src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int BOUND_OF_SIDES = 100;
+    private static final int DEFAULT_RADIUS_OF_CIRCLE = 10;
+    private static final int INDEX_OF_FIGURE = 5;
+    private static final Random RANDOM = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figure[] figuresPool = {
```

Comment:
The approach you use for random figure generation is not optimal, instead of creating one object you create 5. I suggest to use switch/case here